### PR TITLE
GCI-7  Moved freetext from jsp files

### PR DESCRIPTION
**Pull request by itatriev**
[Melange Task](http://www.google-melange.com/gci/task/view/google/gci2014/4663515050672128)

### CODE REVIEW

File: omod/src/main/webapp/uploadForm.jsp [hunk 1]
```diff
@@ -162,7 +162,7 @@ h4 {
 						</td>
 					</tr>
 					<tr>
-						<td>Report Name*</td>
+						<td>'<spring:message code="birt.reportName"/>'</td>
```
Comment (robbyoconnor) [inline omod/src/main/webapp/uploadForm.jsp]: This shouldn't be wrapped in single-quotes....

Comment (itatriev) [inline omod/src/main/webapp/uploadForm.jsp]: @robbyoconnor there are free-texts with quotation marks and without. So, as i think it's different. I read that it's a good practice to use a different style of quotation marks when there are 2 quotes...

Comment (robbyoconnor) [inline omod/src/main/webapp/uploadForm.jsp]: @itatriev Why is it needed?

**Comment by robbyoconnor** [comment]
@itatriev almost there -- Except I just wanted the quotes removed from that ONE instance...not everything...

**Comment by robbyoconnor** [comment]
Approved as part of [GCI 2014](http://www.google-melange.com/gci/task/view/google/gci2014/4663515050672128)

**Pull request merged**